src/config.rs

```diff
@@ -0,0 +1,215 @@
+use std::env;
+use std::fs;
+use std::io::{Error as IoError, Read, Write};
+use std::path::PathBuf;
+
+use ansi_term::{Color, Style};
+use toml;
+use xdg::BaseDirectories;
+
+use error::TealdeerError::{self, ConfigError};
+
+pub const SYNTAX_CONFIG_FILE_NAME: &'static str = "syntax.toml";
+
+#[derive(Copy, Clone, Debug, Serialize, Deserialize, PartialEq, Eq, PartialOrd, Ord)]
```

Comment:
Can you add a `#[serde(rename_all = "lowercase")]` annotation? Then we can put lowercase color names in the configuration file. Capitalized colors seems strange, since everything else is lowercase.

Reply:
Good idea, done.